system_test/common/helpers/aest_nodes.erl

```diff
@@ -122,7 +123,7 @@
         if (buff != \"\") print buff
         buff = $0
     }
-    /^Error in process <[0-9.]*> on node epoch@localhost with exit value:$/ {
+    /^Error in process <[0-9.]*> on node aeternity@localhost with exit value:$/ {
```

Comment:
Does this mean that an old node, if used in a system test, will see its log entries reported as errors? Maybe:
* Need for this line shall be re-assessed. Or
* Check on the node name in this line shall be less strict. Or
* All tests using an old node shall be reviewed for not failing intermittently because of this check e.g. by not validating logs (fragile).